DatadogInternal/Sources/Context/DatadogContext.swift

```diff
@@ -186,9 +186,15 @@ public protocol AdditionalContext {
 extension DatadogContext {
     /// Gets an additional context value.
     ///
-    /// - Parameter type: The value type.
+    /// - Parameter type: The additional context type.
     /// - Returns: The `Context` if found
     public func additionalContext<Context>(ofType type: Context.Type = Context.self) -> Context? where Context: AdditionalContext {
```

Comment:
nit/ I believe we could win if removing the default `= Context.self`. With the default, callers can write:
```swift
let rum: RUMCoreContext = context.additionalContext()
```
which raises questions like _"Is additionalContext() already returning RUMCoreContext?"_ and _"Why repeat the type annotation?"_.

Making the type explicit keeps intent clear and self-documenting:
```swift
let rum = context.additionalContext(ofType: RUMCoreContext.self)
```

Reply:
Done 🙂